content/en/docs/ambient/architecture/control-plane/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Ambient and the Istio control plane
+description: Understand how ambient interacts with the Istio control plane.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+Ambient uses the Istio {{< gloss >}}control plane{{< /gloss>}}, just like other Istio {{< gloss >}}data plane{{< /gloss >}} modes (such as {{< gloss >}}sidecar{{< /gloss >}}). In ambient, rather than the control plane communicating with sidecars in each pod, it communicates with the {{< gloss >}}ztunnel{{< /gloss >}} on each Kubernetes node.
```

Comment:
probably don't need to imply that there are more than 2 modes ....

"..., just like the sidecar mode."

Reply:
Yeah I dithered on this. If we _do_ make mention of other modes in these docs tho, this is where it makes the most sense to.

I'll reword this.